frontend/src/components/Selectors/FilterMethodSelector.tsx

```diff
@@ -0,0 +1,26 @@
+import { chain } from 'lodash'
+import { DropDownMenu } from 'material-ui/DropDownMenu'
+import MenuItem from 'material-ui/MenuItem'
+import * as React from 'react'
+import { SelectorMethod } from '../../constants/datatypes'
+
+type Props = {
+  className?: string
+  onChange(format: SelectorMethod): void
+  value: SelectorMethod
+}
+
+let options: SelectorMethod[] = ['All', 'County Name', 'County Type']
+let menuItems = chain(options).map(
+  _ => <MenuItem value={_} key={_} primaryText={_} />
+).value()
+
+export let FilterMethodSelector: React.StatelessComponent<Props> = ({ className, onChange, value }) =>
```

Comment:
starting to wonder if we should pull this into it's own file and just pass in menu options too and reuse across all dropdowns. seems a little copypasta-y

Reply:
Will add a todo, I also thought about that but it's a bit unclear that we would gain much from abstracting more

Reply:
@akegan Added TODO.